zipkin-cassandra/src/test/scala/com/twitter/zipkin/storage/cassandra/CassieSpanStoreTest.scala

```diff
@@ -15,26 +15,320 @@
  */
 package com.twitter.zipkin.storage.cassandra
 
+import com.datastax.driver.core.Cluster;
 import com.twitter.app.App
-import com.twitter.cassie.tests.util.FakeCassandra
+import com.twitter.conversions.time._
+import com.twitter.util.Await
 import com.twitter.zipkin.cassandra.CassieSpanStoreFactory
+import com.twitter.zipkin.common._
+import com.twitter.zipkin.query.Trace
 import com.twitter.zipkin.storage.util.SpanStoreValidator
-import org.scalatest.FunSuite
+import org.scalatest.{BeforeAndAfter, FunSuite}
+
 
 class CassieSpanStoreTest extends FunSuite {
```

Comment:
currently trying to work around our famous `org.apache.thrift:libthrift:0.5.0-1` vs `0.9.2` runtime compat story.